resources/Components/Monitor/index.js

```diff
@@ -191,85 +205,98 @@ class GasSummaryComponent extends Component {
-          <GasSummary chainId={chainId} displayFeeMarket={displayFeeMarket} gasPrice={gasPrice} />
-        </div>
-        {this.state.expanded ? (
-          <div className='sliceGasBlock'>
-            {displayFeeMarket ? (
-              <GasFeesMarket gasPrice={gasPrice} fees={fees} />
-            ) : (
-              <GasFees gasPrice={gasPrice} />
-            )}
-          </div>
-        ) : null}
-      </div>
+      <>
+        <ClusterRow>
+          <ClusterValue
+            onClick={() => {
+              this.setState({ expanded: !this.state.expanded })
+            }}
+          >
+            <div className='sliceTileGasPrice'>
+              <div className='sliceTileGasPriceIcon' style={{ color: this.props.color }}>
+                {svg.gas(12)}
+              </div>
+              <div className='sliceTileGasPriceNumber'>{gasPrice || '‹0.001'}</div>
+              <div className='sliceTileGasPriceUnit'>{'gwei'}</div>
+            </div>
+          </ClusterValue>
+          <ClusterValue
+            style={{ minWidth: '70px', maxWidth: '70px' }}
+            onClick={
+              explorer
+                ? () => {
+                    if (address) {
+                      link.send('tray:openExplorer', currentChain, null, address)
+                    } else {
+                      link.rpc('openExplorer', currentChain, () => {})
+                    }
+                  }
+                : undefined
+            }
+          >
+            <div style={{ padding: '6px', color: !explorer && 'var(--outerspace05)' }}>
+              <div>{address ? svg.accounts(16) : svg.telescope(18)}</div>
+            </div>
+          </ClusterValue>
+        </ClusterRow>
+        {this.state.expanded && (
+          <ClusterRow>
+            <ClusterValue pointerEvents={true}>
+              <div className='sliceGasBlock'>
+                {displayFeeMarket ? (
+                  <GasFeesMarket gasPrice={gasPrice} fees={fees} color={this.props.color} />
+                ) : (
+                  <GasFees gasPrice={gasPrice} color={this.props.color} />
+                )}
+              </div>
+            </ClusterValue>
+          </ClusterRow>
+        )}
+        <ClusterRow>
+          {this.feeEstimatesUSD({ chainId, displayFeeMarket, gasPrice }).map((estimate, i) => {
+            return (
+              <ClusterValue key={i}>
+                <div className='gasEstimate'>
+                  <div className='gasEstimateRange'>
+                    <span className='gasEstimateSymbol'>
+                      {!estimate.low || estimate.low >= 0.01 ? `$` : '<$'}
+                    </span>
+                    <span className='gasEstimateRangeLow'>{`${
+                      !estimate.low
+                        ? 0
+                        : estimate.low < 0.01
+                        ? 0.01
+                        : estimate.low < 1
+                        ? estimate.low.toFixed(2)
+                        : estimate.low
```

Comment:
would be nice to un-nest this

Reply:
yeah, will need to be in the future unfortunately, this pr is mostly just moving these things, internal refactor will come later